lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -261,19 +276,27 @@ def render_options
             format = " #{format}"
 
             message += sprintf(format, CHECKBOX_ICON[is_chosen]) if @multiple && num && num > 0
-            message += choice.split("\n").map { |l| sprintf(format, l) }.join("\n")
+            message += format_choice(format, choice)
 
             if num == @active
-              message = message.split("\n").map.with_index do |l, idx|
-                idx == 0 ? "{{blue:> #{l.strip}}}" : "{{blue:>#{l.strip}}}"
-              end.join("\n")
+              message = message.split("\n").map { |l| "{{blue:> #{l.strip}}}" }.join("\n")
```

Comment:
This is simplified to ensure that the calculations made in `num_lines` will be correct on lines after the first.